crates/rollup-boost/src/client/rpc.rs

```diff
@@ -474,7 +474,7 @@ pub mod tests {
     use assert_cmd::Command;
     use http::Uri;
     use jsonrpsee::core::client::ClientT;
-    use parking_lot::Mutex;
+    use std::sync::Mutex;
```

Comment:
nit: We should tokio mutexes here.

```suggestion
    use tokio::sync::Mutex;
```